bindings/wasm/examples/worker.ts

```diff
@@ -30,6 +30,7 @@ interface WorkerStatic extends ManifoldStatic {
   cleanup(): void;
 }
 
+//@ts-ignore
```

Comment:
Were you getting an error here?

Reply:
The CI errored out yea (see the first commit of the PR)

Reply:
No, that's from the missing `?` below.

Reply:
Hmm well the CI passed once that comment was added